apps/icons/src/Settings24.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { SVGProps } from 'react';
+const SvgSettings24 = (props: SVGProps<SVGSVGElement>) => (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		width="1em"
+		height="1em"
+		fill="none"
+		viewBox="0 0 24 24"
+		{...props}
+	>
+		<path
+			fill="#758F9E"
```

Comment:
ideally we want this to be `currentColor` so we can set the color via `className` - I noticed it's a bit of a mixed bag in icons right now

Reply:
hmm I could've sworn I made that change and it didn't get included here... will fix this, thanks for the call out